client.go

```diff
@@ -386,6 +390,26 @@ func (s *Client) LoopOut(globalCtx context.Context,
 	}, nil
 }
 
+// getExpiry returns an absolute expiry height based on the sweep confirmation
+// target, constrained by the server terms.
+func (s *Client) getExpiry(height int32, terms *LoopOutTerms,
+	confTarget int32) (int32, error) {
+
+	cltvDelta := confTarget
+
+	switch {
+	case confTarget < terms.MinCltvDelta:
+		cltvDelta = terms.MinCltvDelta
+
+	case confTarget > terms.MaxCltvDelta:
+		return 0, fmt.Errorf("confirmation target %v exceeds maximum "+
```

Comment:
Alternatively we could just clamp to max just like in the min case?

Reply:
That is what I first had, but realized that we also need to protect users against sweeping with very high conf targets while the expiry is much sooner